### DISCUSSION TITLE
Migrate Installation and Database

### DISCUSSION THREAD

**Discussion by ghost** [atrocore/atrocore#671] [Q&A] [answered]
Hay guys,

what is the "correct"/easiest way to migrate the complete system (code, database, configuration, etc) to a new server? 
I have a few paid modules as well which seem to be linked to a system id, dont want to lose the linking. 

Thank you very much

**Comment by atrolex** [comment, marked as answer]
We have it in a classic way:

1) transfer DB by creating a DB dump
2) copy all files to new location
3) ensure all files and scripts have a proper owner and group
4) configure the cron job for a proper user (eg www-data)

If you transfer the system to a new server your system ID will remain, so there is no impact on paid modules.
